style.css

```diff
@@ -5,76 +5,339 @@
 
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap');
 
+.header {
+  display: flex;
+  align-items: center;
+  padding: 1.2rem;
+  margin-left: 0.5rem;
+  justify-content: space-between;
+}
+
+.toolBar {
+  color: #6070ff;
+  font-size: 1.5rem;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  line-height: 20px;
+}
+
+.firstSec {
+  display: flex;
+  flex-direction: column;
+  padding: 114px 24px;
+  width: 100%;
+}
+
+.intoHead {
+  color: #172b4d;
+  font-size: 40px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  line-height: 52px;
+  margin-bottom: 12px;
+  width: 80%;
+}
+
+.intoDes {
+  color: #344563;
+  font-size: 1.1rem;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  line-height: 24px;
+  margin-bottom: 12px;
+  width: 85%;
+}
+
+.contact {
+  color: #7f8cff;
+  font-size: 0.8rem;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  line-height: 20px;
+  word-wrap: break-word;
+  margin-bottom: 12px;
+}
+
+.social {
+  margin-top: 0.5rem;
+}
+
+.social > a {
+  margin-right: 1rem;
+  font-size: 1.5rem;
+  color: #505f79;
+}
+
+.vt:hover {
+  border: 1px solid black;
+  padding: 2px;
+}
+
+.social i:hover {
+  color: black;
+}
+
+.works {
+  display: grid;
+  gap: 88px;
+  padding: 114px 24px;
+}
+
+.cards h2 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 12px 0 0 0;
+}
+
+.brief-list {
+  font-family: 'Poppins', sans-serif;
+  color: #7a869a;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin: 12px 0 0 0;
+}
+
+.self-intro {
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.self-intro p {
+  text-align: justify;
+}
+
+.lang-list {
+  display: flex;
+  color: #6070ff;
+  font-size: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.36px;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.lang-list > li {
+  font-family: 'Poppins', sans-serif;
+  padding: 4px 12px;
+  background-color: #ebebff;
+  margin: 4px;
+  border-radius: 8px;
+}
+
+.btn {
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.51px;
+  margin: 12px 169px 12px 0;
+  padding: 12px;
+  border-radius: 8px;
+  border: 1px solid  #6070ff;
+  color: #396df2;
+  user-select: none;
+  cursor: pointer;
+}
+
+.btn:hover {
+  color: white;
+  background-color: #6070ff;
+}
+
+.aboutMe {
+  color: #172b4d;
+  font-size: 40px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  line-height: 52px;
+  margin-bottom: 12px;
+  width: 80%;
+}
```

Comment:
- [ ] In your about section you should implement the gap between border and social icons  because of your social icons is very close to border and also the spacing of between languages and get my resume button and if you available you should implement the background for about section to align figma design and check out the following images.

**Your Design**

![image](https://github.com/mioOleen/Portfolio-1/assets/118582316/6536863b-1844-4a2c-be5e-5cee79e3fb0d)

**Figma Design**

![image](https://github.com/mioOleen/Portfolio-1/assets/118582316/9bb5512e-8dce-4cc4-a4db-5d1b854d5cdb)